src/tonic/api/points_internal_api.rs

```diff
@@ -0,0 +1,44 @@
+use tonic::{Request, Response, Status};
+
+use crate::tonic::api::points_common::upsert;
+use api::grpc::qdrant::points_internal_server::PointsInternal;
+use api::grpc::qdrant::{PointsOperationResponse, UpsertPointsInternal};
+use std::sync::Arc;
+use storage::content_manager::toc::TableOfContent;
+
+pub struct PointsInternalService {
```

Comment:
Maybe a doc comment that this is intended for P2P communication between peers as part of remote shard functionality?